test/functional/helpers.lua

```diff
@@ -430,6 +466,20 @@ local function create_callindex(func)
   return table
 end
 
+-- Helper to skip tests. Returns true in Windows systems.
+-- pending_func is the pending() from busted
+local function pending_win32(pending_func)
+  clear()
+  if os_name() == 'windows' then
```

Comment:
Woud it be equivalent to use `uname` here as well? Would it be cheaper/faster? Do we still need `os_name()`?